### ISSUE TITLE
release code for training imagenet

### ISSUE THREAD

**Issue by Haijunlv** [timgaripov/swa#19]
hi, i just use your code to reimplement result of resnet50 swa in imagenet.
Before tryiing with imagnet, i have reimplement result of cifar100 from 81.54% to 83.3%.
Then i try to train with imagnet, but i find it is difficult to match 76.97% in 10 epoch finetune stage, actually i only get 71.4580% with swa in 10 epochs.
in your paper, you use cylic learning rate scedule, Is that the main reason you can get  76.97%?
By the way, in 10 epoch finetune with lr=0.0001 ,  model without swa also can get 76.78%.
It seems swa not get surprising improvement compared to only finetune.
Could you relase code for finetuning imagenet? 
![image](https://user-images.githubusercontent.com/28926237/94580955-17fd3500-02ad-11eb-857e-b488760dbee0.png)

**Comment by izmailovpavel** [comment]
It seems like your learning rate schedule is off since you are getting 71.45% which is worse than what you start with? I'd suggest initializing the model with the fine-tuned version that gives you  76.78% and then tuning the learning rate schedule.
